core/src/Threads/Kokkos_ThreadsExec.hpp

```diff
@@ -20,8 +20,9 @@
 #include <Kokkos_Macros.hpp>
 
 #include <cstdio>
-
+#include <ostream>
```

Comment:
Why did you add `<ostream>`?

Reply:
Because it is needed by `Kokkos::Impl::ThreadExec::print_configuration(std::ostream&, const bool detail = false);` defined below.

Reply:
Maybe I could just use `<iosfwd>` in this case :smile:

Reply:
See #6482.